modules/swagger-codegen/src/main/java/com/wordnik/swagger/codegen/languages/RetrofitClientCodegen.java

```diff
@@ -0,0 +1,190 @@
+package com.wordnik.swagger.codegen.languages;
+
+import com.wordnik.swagger.codegen.*;
+import com.wordnik.swagger.models.Operation;
+import com.wordnik.swagger.models.properties.*;
+
+import java.util.*;
+import java.io.File;
+
+public class RetrofitClientCodegen extends DefaultCodegen implements CodegenConfig {
```

Comment:
I doubt how to include additional properties like invokerPackage, groupId, etc.. I think it should be specified in each codegen separately, but I want to avoid copy-pasted solution. 

Also, there are lot of methods that use class fields like sourceFolder which can't be overridden. I hope there'll not become an issue if someone wants to use structure different from "src/java/main/my/favorite/package", it's very rare case, and shouldn't be covered by codegen. Anyway, user can set this field in deriver constructor